person_signup.md

```diff
@@ -135,6 +146,16 @@ OSDI-API-Token:[your api key here]
         "volunteer",
         "donor"
     ],
+    "add_answers": [
+      {
+        "question": "c945d6fe-929e-11e3-a2e9-12313d316c29",
+        "responses": [ "r1", "r2", "r2"]
+      },
+      {
+        "question": "c945d6fe-929e-11e3-a2e9-12313d316c2a",
+        "value": "Heard about the candidate at the Labor Day Picnic"
+      }
```

Comment:
Vis-a-vis the discussion in the other PR, we could simplify this quite a bit as:

```
add_answers: [
  "https://osdisamplesystem.org/api/v1/questions/1234/answers/999",
  "https://osdisamplesystem.org/api/v1/questions/2345/answers/888"
]
```

... if we settle on using the resource path.